src/apps/concepts/components/NamesTable.tsx

```diff
@@ -144,7 +148,13 @@ const NamesTable: React.FC<Props> = ({
                       >
                         {NAME_TYPES.map(nameType => (
                           <MenuItem key={nameType.value} value={nameType.value}>
-                            {nameType.label}
+                            {nameType.label.includes("Synonym") ? (
+                              <div className={classes.menuItemSynonym}>
+                                ({nameType.label})
+                              </div>
+                            ) : (
+                              <div>{nameType.label}</div>
+                            )}
```

Comment:
Two simplifications:

```suggestion
                            <div className={nameType.label === "Synonym" && classes.menuItemSynonym}>
                              ({nameType.label})
                            </div>
```

1. We only want this to apply to "Synonym" not "Custom Synonym" (since presumbably that needs some kind of value).
2. We can do this with a single `div`

Reply:
Hi @ibacher, I tried this with a single div but ended up with an error that says: Type 'string | false' is not assignable to type 'string | undefined'.   Type 'false' is not assignable to type 'string | undefined'.

Reply:
Try this expression instead:

```ts
nameType.label === "Synonym" ? classes.menuItemSynonym : undefined
```

Reply:
@ibacher , it's updated please have a look.